### DISCUSSION TITLE
Allow deletion of Free Orders / Dates with only Free Orders

### DISCUSSION THREAD

**Discussion by psuet** [pretix/pretix#2272] [Ideas]
Hi,

We also use Pretix as a registration tool for appointments (Corona etc.), so there are many dates within pretix.

We want to be able to delete these "appointments" / orders to follow the principles of data minimization.

Currently, we achieve this via the REST API:
We fetch any transaction for dates more than x days in the past and replace all their contact details with "DELETED" (or similar). Not nice and inefficient.

**Comment by raphaelm** [comment]
Full deletion of orders will likely never happen in pretix.

We already provide a tool for bulk anonymization, although the one included in pretix core isn't well-suited for event series admittedly and we should probably fix that at some point.
